### DISCUSSION TITLE
Duty Cycle & Visibility

### DISCUSSION THREAD

**Discussion by 12-uhr-mittags** [OpenCCU/OpenCCU#1403] [🇩🇪 Ideen]
Hallo,
ich sehe im System den Wert des Duty Cycle.
Mein aktuelles Problem ist, dass seit ca. 2 Wochen der Duty Cycle über 100% geht und dass ich keine Idee habe, woran das Liegt. Dieser Wert wird ja durch etwas erstellt.
Da wäre doch eine zusätzliche Option in den Systemeinstellungen hilfreich, wo ich sehen kann was die Ursache ist, wenn der Wert aus dem Ruder läuft. 
Dann könnte ich das sofort abstellen. So tappe ich die ganze Zeit im Dunkeln un treibe Jugend forscht.
Oder habe ich etwas übershen was ich wissen sollte?

**Comment by akula3** [comment]
der [AskSin Analyser](https://asksinpp.de/) kann dir helfen

**Reply by 12-uhr-mittags** [reply]
Ich habe gerade bei "verdrahtet" eine Lösung gefunden die auch gut beschrieben ist.
Aber wenn das so funktioniert, warum gibt es das nicht fertig als Erweiterung zu kaufen ?
Und ... das Problem mit dem Duty Cycle haben noch ganz viele und immer mehr.
